docs/api/navigators/TabNavigator.md

```diff
@@ -85,6 +85,7 @@ The route configs object is a mapping from route name to a route config, which t
 - `tabBarPosition` - position of the tab bar, can be `'top'` or `'bottom'`
 - `swipeEnabled` - whether to allow swiping between tabs
 - `animationEnabled` - whether to animate when changing tabs
+- `configureTransition` - a function that returns a configuration object that describes the animation between tabs
```

Comment:
shouldn't this prop be also added to the props of `TabBarBottom.js` and `TabBarTop.js`?